src/components/ScoreBoard/Header/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const Header = ({title, totalPlayers}) => {
+  return (
+    <header>
+      <h1>{ title }</h1>
+      <span className="stats">Players: {totalPlayers}</span>
+    </header>
+  );
```

Comment:
If we don't have a logic before the `return`, we can remove the curly body brackets and `return` -- the return will still be implied. Similar to this: `const Header = ({ title }) => <h1>{title}</h1>;`. (This applies to other components as well.)